recipes/kmergenie/meta.yaml

```diff
@@ -0,0 +1,32 @@
+about:
+  home: http://kmergenie.bx.psu.edu/
+  license: ''
+  summary: 'KmerGenie estimates the best k-mer length for genome de novo assembly'
+
+build:
+  number: 0
+
+package:
+  name: kmergenie
+  version: '1.7016'
+
+requirements:
+  build:
+      - gcc
+      - zlib
+      - r-base
+      - r-essentials
+      - python 2.7.12
```

Comment:
like here for example: https://github.com/bioconda/bioconda-recipes/blob/60d2ea750430dbe0191f6f8d5639f90fc1223bd9/recipes/pynast/1.2.2/meta.yaml#L12